src/main/java/com/autilus/hw0207/service/AccountService.java

```diff
@@ -0,0 +1,11 @@
+package com.autilus.hw0207.service;
+
+import lombok.Data;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Data
+public class AccountService {
+    private Map<String, Object> data = new HashMap<>();
```

Comment:
because ```.getPage``` method in ```SignInServlet``` require ```Map<String, Object>```

<img width="662" alt="Screen Shot 2019-07-02 at 6 31 12 PM" src="https://user-images.githubusercontent.com/25722810/60525850-da9ba780-9cf7-11e9-831f-290e2e319180.png">

Reply:
ok